kubernetes/loculus/templates/loculus-preprocessing-config.yaml

```diff
@@ -5,6 +5,9 @@
 {{- $nucleotideSequencesList := (eq $rawNucleotideSequences nil | ternary (list "main") $rawNucleotideSequences) }}
 {{- range $processingIndex, $processingConfig := $organismConfig.preprocessing }}
 {{- if $processingConfig.configFile }}
+{{- /* Use the enaDepositionConfig as the base config for scientific_name and molecule_type */}}
```

Comment:
We should probably only import these fields if the enaDeposition container is also configured e.g. `{{- if not .Values.disableEnaSubmission }}`

Reply:
There's no benefit to making it conditional on not being disabled. Disabling shouldn't change the flatfile.